ClientApp/src/App.scss

```diff
@@ -0,0 +1,10 @@
+@import './Styles/constants';
```

Comment:
I genuinely have no idea, but do we need this import here as well as all the individual pages?